phoenix-core/src/it/java/org/apache/phoenix/end2end/CreateTableIT.java

```diff
@@ -90,6 +91,66 @@ public void testStartKeyStopKey() throws SQLException {
         assertTrue(splits.size() > 0);
     }
 
+    @Test
```

Comment:
Can you add a test for adding the identical column with an alter table statement ?

Reply:
Done in the same test